plugin/aws/s3Bucket.go

```diff
@@ -146,23 +146,24 @@ func listObjects(ctx context.Context, client *s3Client.S3, bucket string, prefix
 // s3Bucket represents an S3 bucket.
 type s3Bucket struct {
 	plugin.EntryBase
-	ctime   time.Time
+	crtime  time.Time
 	client  *s3Client.S3
 	session *session.Session
 }
 
-func newS3Bucket(name string, ctime time.Time, session *session.Session) *s3Bucket {
+func newS3Bucket(name string, crtime time.Time, session *session.Session) *s3Bucket {
 	bucket := &s3Bucket{
 		EntryBase: plugin.NewEntry(name),
 	}
-	bucket.ctime = ctime
+	bucket.crtime = crtime
 	bucket.client = s3Client.New(session)
 	bucket.session = session
 	bucket.
 		Attributes().
-		SetCtime(bucket.ctime).
-		SetMtime(bucket.ctime).
-		SetAtime(bucket.ctime)
+		SetCrtime(bucket.crtime).
+		SetMtime(bucket.crtime).
```

Comment:
Should we bother setting the rest? We don't have any specific information.

Reply:
Don't know. It's a pattern we've been using across multiple things which is why I replicated it here. Personally I prefer we use the information we have, so the answer would be "No." What about you?